File: User_Input.py
def main():
    lines = []
    print("Enter lines of text. Type 'print' to display all entries.")

    while True:
        user_input = input()
        if user_input == "print":
            print("Stored lines:")
            for line in lines:
                print(line)
            break
        lines.append(user_input)

if __name__ == "__main__":
    main()

File: User_Input.cpp
#include <iostream>
#include <vector>
#include <string>

int main() {
    std::vector<std::string> lines;
    std::string input;

    std::cout << "Enter lines of text. Type 'print' to display all entries.\n";

    while (true) {
        std::getline(std::cin, input);

        if (input == "print") {
            std::cout << "Stored lines:\n";
            for (const auto& line : lines) {
                std::cout << line << std::endl;
            }
            break;
        }

        lines.push_back(input);
    }

    return 0;
}
